### ISSUE TITLE
KP.3.1 got again S:S31del and Orf1a:N4256S (NSP10_N3S) ( 44 seqs, >10 places)

### ISSUE THREAD

**Issue by FedeGueli** [sars-cov-2-variants/lineage-proposals#1770]
KP.3.1 acquired again s:S31del and curiously also this time, AS KP.3.1.1, it is accompanied by a NSP10 mutation that will make possible to track the lineage. it is already in three countries so we will see if it will be as fast as the "Leading cousin"

KP.3.1>> Orf1a:N4256S (NSp10_N3S), S:S31del (Del21653-55)
Query:A13032G,del21653
Samples:3 (EPI_ISL_19259541, EPI_ISL_19261998, EPI_ISL_19280024) 
Country: Italy (First sample collected on 2024-06-26), France (via GBW) , Netherlands

Tree:
<img width="1152" alt="Screenshot 2024-07-24 alle 08 41 47" src="https://github.com/user-attachments/assets/0813f9b7-bdff-4792-b4ed-040e6a410b92">
https://nextstrain.org/fetch/genome-test.gi.ucsc.edu/trash/ct/subtreeAuspice1_genome_test_1d7d9_9fb80.json?c=userOrOld&label=id:node_11557721

**Comment by FedeGueli** [comment]
+1 IT

**Comment by FedeGueli** [comment]
likely dead closing this

**Comment by FedeGueli** [comment]
After a hyatus two recent samples of this have been uploaded one from Canada and one very recent from Singapore, this last one has also S:Q271E a C>G mutation.
Reopening it for a while.

**Comment by FedeGueli** [comment]
+1 Germany

**Comment by FedeGueli** [comment]
11 US

**Comment by FedeGueli** [comment]
14, Poland

**Comment by FedeGueli** [comment]
Slowed down again closing this

**Comment by FedeGueli** [comment]
16, Canary Island

**Comment by FedeGueli** [comment]
19, striking how sloe it is this one in comparison to Kp.3.1.1

**Comment by DailyCovidCases** [comment]
Slowing down now

**Comment by FedeGueli** [comment]
> A13032G,del21653

yeah it has been superslow, i m keeping it open due to september 6 seqs

**Comment by FedeGueli** [comment]
21 GBW Italy

**Comment by FedeGueli** [comment]
22

**Comment by xz-keg** [comment]
23 now also in USA-MN,  slow but stable maybe worth a proposal.

**Comment by FedeGueli** [comment]
24 Spain

**Comment by FedeGueli** [comment]
+1 Ontario

**Comment by FedeGueli** [comment]
+4 Germany, Spain, Italy , Canada

**Comment by FedeGueli** [comment]
two recent samples in Canada

**Comment by FedeGueli** [comment]
+2 Ontario

**Comment by FedeGueli** [comment]
42 cc @xz-keg this is still around despite being slow.

**Comment by xz-keg** [comment]
> 42 cc @xz-keg this is still around despite being slow, thinking to propose it to specify Kp.3+31del lineages.

yeah worth proposal, likely dominated sometime somewhere in Africa.

**Comment by FedeGueli** [comment]
44 but likeluy dying off finally

**Comment by FedeGueli** [comment]
Def dead
